loader-tests/utils.ts

```diff
@@ -1,63 +1,91 @@
+import {
+  enableTestTime,
+  getTestEnvironment,
+  setBaseTestRoute,
+  setRequestMethod,
+  setupEnvironment,
+  userId
+} from '@slowreader/core'
 import type { ReadableAtom } from 'nanostores'
-import { isAbsolute, join, resolve } from 'node:path'
+import { readFile } from 'node:fs/promises'
+import { isAbsolute, join } from 'node:path'
 import pico from 'picocolors'
 
-const supportedExtensions = ['.xml', '.opml']
-
-export function isSupportedExt(path: string): boolean {
-  return supportedExtensions.some(ext => path.endsWith(ext))
+export async function readText(path: string): Promise<string> {
+  let absolute = path
+  if (!isAbsolute(absolute)) {
+    absolute = join(process.env.INIT_CWD!, path)
+  }
+  let buffer = await readFile(absolute)
+  return buffer.toString('utf-8')
 }
 
 export function isString(attr: null | string): attr is string {
   return typeof attr === 'string' && attr.length > 0
 }
 
-export function resolvePath(path: string): string {
-  if (isAbsolute(path)) {
-    return path
-  }
-  return resolve(join(process.cwd(), '..', path))
+export function enableTestClient(): void {
```

Comment:
I moved test environment initializer here since we will use the same in many tests